tempesta_fw/http.c

```diff
@@ -3228,6 +3255,43 @@ tfw_cfgop_resp_body(TfwCfgSpec *cs, TfwCfgEntry *ce)
 	return ret;
 }
 
+static int
+tfw_cfgop_whitelist_mark(TfwCfgSpec *cs, TfwCfgEntry *ce)
+{
+	unsigned int i;
+	const char *val;
+
+	if (!ce->val_n) {
+		TFW_ERR_NL("Invalid number of arguments: %zu\n", ce->val_n);
+		return -EINVAL;
+	}
+	if (ce->attr_n) {
+		TFW_ERR_NL("Unexpected attributes\n");
+		return -EINVAL;
+	}
+
+	tfw_wl_marks.sz = ce->val_n;
+	if (!(tfw_wl_marks.mrks = kmalloc(ce->val_n, GFP_KERNEL)))
+		return -ENOMEM;
```

Comment:
Is there an understanding of how many marks can be specified? If there can be a lot, then it would be convenient to allow repetitive use of this option, where just a subset of marks is specified in each line, for better readability.

Right now `allow_repeat` is set to `false` (which is the default), and the code can't handle multiple specifications of this option.

Reply:
Yes, this question was the subject of discussion and it was decided to use one directive since the number of marks should be supposedly small.